react/README.md

```diff
@@ -0,0 +1,124 @@
+[![Build Status](https://travis-ci.org/sparkdesignsystem/spark-design-system.svg?branch=staging)](https://travis-ci.org/sparkdesignsystem/spark-design-system)
+
+# Spark Design System React
+
+Welcome to the Spark Design System React, a product of [Quicken Loans](https://github.com/quickenloans).
+
+Spark Design System is where we collect our best solutions for how to design and build unified digital experiences for the Rocket brands.
+
+* [Spark](https://www.npmjs.com/package/@sparkdesignsystem/spark) contains the basic patterns and components that make up
+  the visual and interaction design of the Quicken Loans family of
+  Fintech products. If your application is expected to look and behave
+  like a Quicken Loans application, this package is the starting point.
+  When the Spark Sass and JS setup files are imported into an application you will receive code for all of Spark.
+
```

Comment:
This needs updating to reflect the new install process.

Reply:
I'll take care of this!

Reply:
https://github.com/sparkdesignsystem/spark-design-system/pull/3663/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Reply:
DONE!
https://github.com/sparkdesignsystem/spark-design-system/pull/3663/files#diff-dbd8999c1b6d2e969e3d8fbdce0b705d0f72158e344900d3a4fb29b0a3b18beb